Atala PRISM/2.10.md

```diff
@@ -0,0 +1,118 @@
+
+### Documentation
+
+* [Documentation Portal v1.81.0](https://github.com/input-output-hk/atala-prism-docs/releases/tag/v1.81.0)
+
+## Changelog
+
+**Updated components:**
+
+| Component             | From    | To         |
+| --------------------- | ------- | ---------- |
+| Documentation Portal  | v1.78.1 | v1.81.0    |
+| Open Enterprise Agent | v1.25.0 | v1.28.0    |
+| PRISM Mediator        | v0.10.2 | v0.14.1    |
+| PRISM Node            | v2.2.1  | no changes |
+| Wallet SDK KMM        | v2.7.0  | v3.0.0     |
+| Wallet SDK TypeScript | v4.0.2  | v5.0.0     |
+| Wallet SDK Swift      | v4.0.0  | v5.0.0     |
+| Apollo Crypto Library | v1.2.10 | no changes |
+
+### Open Enterprise Agent
+
+#### Breaking changes
+
+- Update `DIDComm PeerDID` library to support latest specs
+
+#### New features
+
+- Add credential definition performance test
+- Add Json VC schema meta validation
+- Support vault AppRole authentication
+- Interoperable schema changes
+
+#### Bug fixes 
+
+- Increase http timeout communication channel closing
+- Re-enable logging with SLF4J and add traceId
+- Remove OAS schema format for empty response body
+- Remove hard code `did:example:*`
+
+### Documentation Portal
+
+- Update the Agent API to OEA `v1.28.0` version
+
+### Wallet SDK TypeScript
+
+#### Breaking changes
+
+- `Domain.Credential` now has uuid (string) as an optional attribute in the class
+- Change `StorePrismDID` Function parameters, removing keyPathIndex and privateKeyMetadataId which were not in use
+- Small change in the `getCredentialMetadata` and `getLinkSecret` to return `null` when not found
+- Change `Pluto` interface to use `CredentialMetadata` class VS what it was using before.
+- Change `Pluto` interface, `storeMediator` now accepts a `Mediator` class and not 3 attributes, mediator, host and routing
+- Rename `storePrivateKeys` to `storePrivateKey`
+- Change `Pluto` interface, using `Domain.PrismDID` instead of `Domain.PrismDID` which is being removed
+- Added new properties to handle Anoncreds credentials properties.
+- Re-implemented `Pluto` which is now available again to all the users.
+
+```typescript
+import SDK from "@atala/prism-wallet-sdk";
+import IndexDB from "@pluto-encrypted/indexdb";
+
+const store = new SDK.Store({ name: "test", storage: IndexDB, password: Buffer.from("demoapp").toString("hex") });
+const pluto = new SDK.Pluto(store, apollo);
+```
+
+Migrations, schemas, queries and error handling have been moved to the SDK again but user's which were using the existing storages can just pass the indexDB pluto-encrypted storage and it will integrate with the new db schemas, migrations, etc
+
+#### New features
+
+- Live mode using `websockets`
+
+#### Bug fixes
+
+- Key's id name according to the DID Peer new specs
+- Recover `JTI` field correctly. Allowing to regenerate the original `JWT` string
+
+### Wallet SDK KMM
+
+#### Breaking changes
+
+- `PeerDID` specification changes
```

Comment:
Can we precise the spec version? Or a link to the spec?

Reply:
@FabioPinheiro can you confirm this is the spec we are follwing?
https://github.com/decentralized-identity/peer-did-method-spec/blob/master/spec/core.md#generating-a-didpeer2

Reply:
yes

Basically, the specification was updated to remove some ambiguities. In order for different implementations to be interoperable. But these caused breaking change in all the implementations that existed of the did Peer.

We can mention for further reading check the link https://github.com/decentralized-identity/peer-did-method-spec/pull/62